examples/GaussExample/mainExampleGauss.m

```diff
@@ -27,7 +27,6 @@
 end
 
 opt                    = PestoSamplingOptions();
-opt.rndSeed            = 3;
```

Comment:
You always have a certain chance that for some freak random seed the methods on the examples are not behaving as expected. I suppose that for sampling the risk is higher (you could make a large step in the beginning and just happen to accept it), but I would anyways initialize all examples with a random seed.